src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -0,0 +1,173 @@
+@import "../../patternfly-utilities";
+
+// The value of component scoped variables is always defined by a global variable:
+.pf-l-toolbar {
+
+  /* stylelint-disable */
+
+  // Toolbar Section
+  --pf-l-toolbar__section--MarginTop:           var(--pf-global--spacer--md);
+  --pf-l-toolbar__section--PaddingTop:          var(--pf-global--spacer--md);
+  --pf-l-toolbar__section--PaddingBottom:       var(--pf-global--spacer--md);
+  --pf-l-toolbar__section--BorderColor:         var(--pf-global--BorderColor);
+  --pf-l-toolbar__section--BorderTopWidth:      var(--pf-global--BorderWidth--sm);
+  --pf-l-toolbar__section--BorderBottomWidth:   var(--pf-global--BorderWidth--sm);
+
+  // Toolbar Group
+  --pf-l-toolbar__group--MarginRight: var(--pf-global--spacer--xl);
+  --pf-l-toolbar__group--MarginLeft:  var(--pf-global--spacer--xl);
+
+  // Toolbar Item
+  --pf-l-toolbar__item--MarginRight:  var(--pf-global--spacer--md);
+  --pf-l-toolbar__item--MarginLeft:   var(--pf-global--spacer--md);;
+
+  /* stylelint-enable */
+
+  // Toolbar base styling
+  flex-flow: row wrap;
```

Comment:
If the toolbar isn't supposed to wrap by default, and you apply `.pf-m-wrap` for it to wrap, should this be `nowrap` instead (or just remove the entire line since `row` and `nowrap` are defaults)

Reply:
You could also just use `flex-wrap: wrap` here.